x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_ecommerce/ml/high_sum_total_sales.json

```diff
@@ -0,0 +1,39 @@
+{
+  "groups": ["kibana_sample_data"],
+  "description": "Kibana sample eCommerce data : detect anomalies in total sales",
```

Comment:
Convention for other modules has been not to have a space before the colon `:` character - should just be `Kibana sample eCommerce data: detect anomalies in total sales`.